File: Baekjoon/python/8371.py
# source : https://www.acmicpc.net/problem/8371

def main():
    # input
    n = int(input())
    a, b = input(), input()
    
    result = 0
    for i in range(n):
        if a[i] != b[i]:
            result += 1
    
    print(result)

if __name__ == '__main__':
    main()

File: Baekjoon/typescript/8371.ts
// source : https://www.acmicpc.net/problem/8371

// input
const input: Array<string> = require('fs').readFileSync('example.txt').toString().replace(/\r/g, "").split('\n');

const [n, a, b] = input;

let result = 0;
for (let i = 0; i < Number(n); i++) {
    if (a[i] !== b[i]) {
        result++;
    }
}

console.log(result);